blog/_posts/2025-04-24-network-isolated-cluster.md

```diff
@@ -9,7 +9,7 @@ categories:
 
 ## Network isolated Azure Kubernetes Service (AKS) cluster
 
-You may have heard about [AKS network isolated cluster](https://learn.microsoft.com/azure/aks/concepts-network-isolated) during Microsoft last year. This feature allows you to bootstrap an AKS cluster without any outbound egress traffic at a glance. This feature is now Generallly available. With this announcement, we want to share with you more details of this exciting offering. 
+You may have heard about [AKS network isolated cluster](https://learn.microsoft.com/azure/aks/concepts-network-isolated) during Microsoft last year. This feature allows you to bootstrap an AKS cluster without any outbound egress traffic at a glance. This feature is now Generally available. With this announcement, we want to share with you more details of this exciting offering. 
```

Comment:
What does "during Microsoft" mean? Did you mean any specific event or announcement?

Reply:
Ops, missed the Build, added it back